bin/sdl.cc

```diff
@@ -452,7 +469,7 @@ void run_sdl()
                 }
             }
 
-            if (ana.do_write_ntuple)
+            if (ana.do_write_ntuple or ana.gnn_ntuple)
```

Comment:
I can see an "or" here, meaning that one can in principle write out an NTuple with only the GNN branches. The changes in lines L307-L311 of the new version of this file corroborate this assumption. However, when I look at L13 of the `write_sdl_ntuple.cc` file, it seems like the regular branches are filled in any case?

Maybe I misinterpret the code but have the different combination of the `do_write_ntuple` and `gnn_ntuple` options been tested and work fine? And, ultimately, what is the intended behavior? Do we want the option to write out only the GNN NTuple and not the rest?

Reply:
I added so that do_write_ntuple must be set to 1 when gnn_ntuple is to be set.
The "or" statements are no longer necessary because of it.